projects/models.py

```diff
@@ -78,6 +92,97 @@ class ProjectExtras(models.Model):
     nickname = models.CharField(max_length=50, blank=False, null=False, unique=True)
 
 
+class InvitationQuerySet(models.QuerySet):
+    pass
+
+
+class InvitationManager(models.Manager):
+    pass
+
+
+class Invitation(models.Model):
+    """Model to hold invitations of users to projects."""
+
+    STATUS = [("ISSUED", "Issued"), ("ACCEPTED", "Accepted")]
```

Comment:
Can you put the values ("ISSUED", "ACCEPTED") as constants, which you can then refer to in other places? This will make it harder to screw up referencing them in some other class/elsewhere. For example, in the celery task.